### ISSUE TITLE
413 Request Entity Too Large for mjml API

### ISSUE THREAD

**Issue by Blastnsmash** [mjmlio/mjml#1758]
**Describe the bug**
Receiving  413 Request Entity Too Large on API

MJML will not render on https://mjml.io/try-it-live unless you clip a little bit of the content. 

**To Reproduce**
Upload mjml content of size over 100 KB 

**Expected behavior**
Should return content

**MJML environment (please complete the following information):**
 - OS: [e.g. MacOS]
 - MJML Version [e.g. 4.2.0]
 - MJML tool used [e.g MJML App]

**Email sending environment(for rendering issues)**:
non applicable

**Affected email clients (for rendering issues):**
non applicable

**Screenshots**
non applicable

**Additional context**
 non applicable

**Comment by ngarnier** [comment]
Can you please provide the payload to reproduce?

**Comment by Blastnsmash** [comment]
https://meridian058.sharepoint.com/:t:/s/MeridianPublic/EU2XfPFpiS1MlVojujt991MBeZX4fU44Jv6z4_MSoODrmQ?e=BnpYxE

Let me know if you're unable to view the link.

**Comment by ngarnier** [comment]
The issue is due to your content being invalid, you can see on the try it live that it's rendered as "MJML badly formatted". Indeed, you can see one error for example (I'm not going to go through all errors) on line 173 as there is one `span` that isn't closed.

I obviously wouldn't recommend anyone to have a MJML file of 100kb as it means the HTML is going to be 10 times that, but you can see for example that here with a 200kb MJML file it's properly rendered: https://mjml.io/try-it-live/r1Thujb3r.

TL;DR: the issue is with your file containing XML errors.

**Comment by Blastnsmash** [comment]
My apologies, the server error received led me in the wrong direction. And, I just clipped a little bit off the end and it rendered - I must of missed the bad format. Thank you for the correction, greatly appreciated.

**Comment by aksld** [comment]
Hello,

I would like to reopen the issue because I have this error message in the try-it-live console.

Fy file is 1947 lines long. This is a lot but necessary for a business logic. Is there a limit? Thanks
